homeassistant/components/ssdp/__init__.py

```diff
@@ -85,6 +88,65 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+@dataclass
+class _HaServiceDescription:
+    """Keys added by HA."""
+
+    x_homeassistant_matching_domains: set[str]
```

Comment:
A default should not be necessary, but it might make it easier when we create the test instances (most component don't use it).
```suggestion
    x_homeassistant_matching_domains: set[str] = field(default_factory=set)
```